dynamo/prediction/least_action_path.py

```diff
@@ -601,7 +601,7 @@ def least_action(
 
     adata.uns[LAP_key] = {
         "init_states": init_states,
-        "init_cells": init_cells,
+        "init_cells": list(init_cells),
```

Comment:
why list only applies to init_cells instead of init_states?

Reply:
`init_cells` can be an Index (so we convert it to a list) while `init_states` is an array (a more compatible data type).